docs/content/preview/develop/best-practices-ysql.md

```diff
@@ -130,3 +130,20 @@ For more details, see [Distributed parallel queries](../../api/ysql/exprs/func_y
 ## TRUNCATE tables instead of DELETE
 
 [TRUNCATE](../../api/ysql/the-sql-language/statements/ddl_truncate/) deletes the database files that store the table and is much faster than [DELETE](../../api/ysql/the-sql-language/statements/dml_delete/) which inserts a _delete marker_ for each row in transactions that are later removed from storage during compaction runs.
+
+
+## Number of tables and indexes
+Each table and index is split into tablets and each tablet has overhead. See [tablets per server](#tablets-per-server) for limits.
+
+## Tablets per server
+Each table and index consists of several tablets based on the [`--ysql_num_shards_per_tserver`](../../reference/configuration/yb-tserver#ysql_num_shards_per_tserver)
+gflag. In a cluster with RF3, 1000 tablets have an overhead of 0.4vcpu for raft heartbeats (assuming 0.5s heartbeat interval), 300MB memory, 128GB disk-space for WAL.
+You have to keep this number in mind depending on the number of tables and number of tablets per-server that you intend to create. Note that each tablet can contain 100GB+ of data. 
+We're actively working on [#1317](https://github.com/yugabyte/yugabyte-db/issues/1317) to increase this limit.
+
+There are different ways to reduce number of tablets:
+
+1. Use [colocation](../../architecture/docdb-sharding/colocated-tables/) to group small tables into 1 tablet.
+2. Reduce number of tablets-per-table using [`--ysql_num_shards_per_tserver`](../../reference/configuration/yb-tserver#ysql_num_shards_per_tserver) gflag.
+3. Use [`SPLIT INTO`](../api/ysql/commands/ddl_create_table.md#split-into) clause when creating the table.
+4. Start with few tablets and use [automatic tablet splitting](../../architecture/docdb-sharding/tablet-splitting/)
```

Comment:
```suggestion
- Start with few tablets and use [automatic tablet splitting](../../architecture/docdb-sharding/tablet-splitting/).
```

Reply:
You don't need to add number them unless they are ordered steps. Regular bullets are fine.